commands/utils/create-terminal.ts

```diff
@@ -0,0 +1,9 @@
+import * as vscode from "vscode";
+import { setEnvInTerminal } from '../change-endpoint';
+
+export async function createTerminal(name: string) : Promise<vscode.Terminal> {
```

Comment:
Why did you make createTerminal async?  Doesn't seem to be required, and changes a bunch of calls.